### DISCUSSION TITLE
announce.dismiss not working when deployed

### DISCUSSION THREAD

**Discussion by goddard82** [squidfunk/mkdocs-material#3525] [Q&A] [answered]
I have insiders access, and when I have the `announce.dismiss` feature in my mkdocs.yml locally, the X to exit the banner appears as expected. 

When this is deployed to the 'dev' version of my mkdocs site, the X doesn't appear. The versions of Material and Insiders are the same locally and deployed on Dev, and there are no differences in any other files.

Is this a known issue, or something missing on my end?

**Comment by squidfunk** [comment, marked as answer]
It's not a known issue and likely related to misconfiguration. Please provide more information, e.g. how you override, how you deploy, how you install Insiders etc. Everything that might be relevant. Ideally you could share the link where it's not working.

**Reply by goddard82** [reply]
Our microsite is currently deployed behind a VPN so I can't share that - would it be useful to share a snippet of the mkdocs.yml file?

Insiders is installed via a Pipfile and docker

**Reply by squidfunk** [reply]
Yes. Anything that would allow me to view at least some portion of how you set it up. I'm afraid I'm only as good as the information you give me, so everything that you can share is valuable. Note that when the announcement bar icon is not shown, it's very likely that your either not building with Insiders, or [didn't correctly enable the feature](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/?h=mark+read#mark-as-read).

To verify whether the documentation you deployed is built with Insiders, you can check the `generator` tag in the `head` of your document, which should show something like:

``` html
<meta name="generator" content="mkdocs-1.2.3, mkdocs-material-8.1.9+insiders-4.7.2">
```

Also, please make sure that you're building your docs with an Insiders version greater than 4.2.0, which is the version in which the feature was added.

**Reply by goddard82** [reply]
The generator tag:
`<meta name="generator" content="mkdocs-1.2.3, mkdocs-material-8.1.9+insiders-4.7.1">`

Some of my mkdocs.yml

```# Theme configuration
theme:
  name: material
  custom_dir: src/overrides
  language: en
  font:
    text: Open Sans
    code: JetBrains Mono
  features:
    - announce.dismiss
    - content.code.annotate
    - navigation.tabs
    - navigation.instant
    - navigation.sections
    - navigation.tracking
    - navigation.top
  palette:
    - scheme: slate
      primary: green
      accent: light green
      toggle:
        icon: material/lightbulb-outline
        name: Switch to dark mode
    - scheme: default
      primary: green
      accent: light green
      toggle:
        icon: material/lightbulb
        name: Switch to light mode
  logo: img/Golem.svg
  icon:
    repo: fontawesome/brands/github
  favicon: img/favicon.png
  static_templates:
    - 404.html
    - offline.html

extra_css:
  - css/extra.css

extra:
#  analytics:
#    provider: google
#    property: !ENV GOOGLE_ANALYTICS_KEY
  # consent:
  #   title: Cookie consent
  #   description: >
  #     We use cookies to recognize your repeated visits and preferences, as well
  #     as to measure the effectiveness of our documentation and whether users
  #     find what they're searching for. With your consent, you're helping us to
  #     make our documentation better.-
  feedback_url: https://github.com/lkwd/golem_docs/issues/new
  generator: false
  robots: !ENV [ROBOTS, "noindex"]
  social:
    - icon: fontawesome/brands/slack
      link: https://lkwd.slack.com/archives/C5BB6373J
      name: Game Framework Slack channel
    - icon: fontawesome/brands/github
      link: https://github.com/lkwd
    - icon: fontawesome/brands/discord
      link: https://discord.gg/avakin-creators
      name: Discord
    - icon: fontawesome/brands/youtube
      link: https://www.youtube.com/channel/UCIUo96FoxYAdJMofyydAn5A
      name: YouTube
    - icon: fontawesome/brands/reddit
      link: https://www.reddit.com/r/AvakinOfficial
      name: Reddit


plugins:
    - search
    - mkdocstrings
#    - tags
##    - schema_reader:
##        include:
##          - "src/"
##          - "src/config.json"

markdown_extensions:
#  - meta
  - attr_list
  - def_list
  - markdown.extensions.admonition
  - pymdownx.inlinehilite
  - pymdownx.tasklist:
      custom_checkbox: true
  - pymdownx.superfences:
      custom_fences:
        - name: mermaid
          class: mermaid
          format: !!python/name:pymdownx.superfences.fence_code_format
  - pymdownx.details
  - markdown.extensions.footnotes
  - pymdownx.highlight:
      linenums: true
      use_pygments: true
  - pymdownx.keys:
      camel_case: true
  - pymdownx.emoji:
      emoji_index: !!python/name:materialx.emoji.twemoji
      emoji_generator: !!python/name:materialx.emoji.to_svg```

**Comment by goddard82** [comment]
This was a mistake on my end, apologies - and thanks for your help!

**Reply by squidfunk** [reply]
Could you explain what you did wrong? It would probably be of help for other users 😊

**Reply by goddard82** [reply]
I had a part of my build process where I use a template for the .yml, which I overwrite with recent changes to the Nav. In this template. the feature was missing 🤦🏼
